src/map/utils/charutils.cpp

```diff
@@ -2866,7 +2858,18 @@ namespace charutils
             skillBonus += PChar->PMeritPoints->GetMeritValue(skillMerit[meritIndex], PChar);
             meritIndex++;
 
-            skillBonus += PChar->getMod(static_cast<Mod>(i + 79));
+            if (i == 44)
```

Comment:
Call me crazy, but I believe the proper fix for Temple Cyclas would be to just edit the item mod entries in the db for that item?

Reply:
the item mod entries are actually correct tho

Reply:
Has someone confirmed this issue with retail client, or is this something happening with an outdated one?

Reply:
I kind of want to swap the mod IDs just to put the skills in order, but that's my OCD talking

Reply:
And I seriously dislike that nothing was making it obvious wtf `i` actually was and had to be inferred

Reply:
lucy our predecessors have some splaining' to do!

Reply:
opinion: 

1. for now, I think we should swap the IDs back that my ocd isn't far off from sane for a change
2. raw magic numbers should get replaced by their skill IDs name from the enumeration
3. a command above `for (int32 i = 1; i < 48; ++i)` should be added to inform editors we're iterating skill IDs and offsetting by 79 to get the modifier ID
4. in the future (and for now a TODO comment) this whole thing should eventually get a refactored to be less dependent on arbitrary ordering of modifier IDs and conditionals on skill ranges.